Assets/Scripts/InfoBoothSetup.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Photon.Pun;
+using UnityEngine;
+
+public class InfoBoothSetup : MonoBehaviourPunCallbacks {
+    private Camera __camera;
+    private EventInfoManager __eventInfoManager;
+
+    void Start() {
+        __eventInfoManager= GameObject.Find(GameConstants.k_EventInfoManager).GetComponent<EventInfoManager>();

```

Comment:
Space after __eventInfoManager

Reply:
Thanks! I'll correct this in the next PR!